### PR TITLE
icon-2021-ingestion 

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Comment by xinru1414** [comment]
waiting on ingestion material for several volumes

**Comment by mjpost** [comment]
Is this still waiting for information?

**Comment by xinru1414** [comment]
@mjpost Yes, still waiting on material for Workshop on Speech and Music Processing 2021 (SMP2021)

**Comment by xinru1414** [comment]
Awaiting updates on two of the workshops.

**Comment by mjpost** [comment]
A few comments:

* volume ordering: the shared task should be listed second
* the three workshops should probably be top-level events, rather than just listed as ICON workshops (or is there a reason you did it this way?)

**Comment by xinru1414** [comment]
@mjpost just checked out [ICON 2020 ](https://aclanthology.org/events/icon-2020/), looks like both shared task and workshop are listed as top-level events. I will follow the convention then.

**Comment by mjpost** [comment]
Okay, I think that was a mistake last year. We can leave that, but can you pull these two out as proper workshops and link them via joint.xml?

File: data/yaml/venues.yaml [hunk 1]
```diff
@@ -798,6 +798,9 @@ nlp4convai:
   is_acl: true
   name: The Workshop on Natural Language Processing for Conversational AI
   url: https://sites.google.com/view/2ndnlp4convai/home
+nlp4dh:
+  acronym: nlp4dh
```
Comment (mjpost) [inline data/yaml/venues.yaml:802]: This should be "NLP4DH"

File: data/yaml/venues.yaml [hunk 2]
```diff
@@ -1055,6 +1061,9 @@ smila:
 smm4h:
   acronym: SMM4H
   name: Social Media Mining for Health Applications Workshop
+smp:
+  acronym: smp
```
Comment (mjpost) [inline data/yaml/venues.yaml:1065]: SMP (caps)

**Comment by xinru1414** [comment]
@mjpost ready for review + merge

**Review by mjpost — APPROVED** [review]

**Pull request merged**